platform/core.startup/src/org/netbeans/core/startup/Splash.java

```diff
@@ -633,7 +637,10 @@ final void addToMaxSteps(int steps) {
         }
 	
         void paint() {
-            image.paintIcon(comp, graphics, 0, 0);
+            // loadContentIcon may return a null image
+            if (image != null) {
```

Comment:
Yes, reviewing the initialization of image, this one can be null if the graphic file is not found. So this is a good check to include. Thanks!